### ISSUE TITLE
Implement statImprintMult and usedStats fields in species exports

### ISSUE THREAD

**Issue by coldino** [arkutils/Purlovia#40]
From Cad's work tracker:

Add optional double-array named `statImprintMult` for custom imprinting multiplier factors. If that value is not available or null, the default of `[0.2, 0, 0.2, 0, 0.2, 0.2, 0, 0.2, 0.2, 0.2, 0, 0]` will be used. Mostly this array is only needed for flyers, which don't level speed.

Add bit-flag-array for `usedStats` per species, saved as int (e.g. a species using only stats with index 0, 1 and 3 will have 11 as value). This makes `doesNotUseOxygen` obsolete.

**Comment by coldino** [comment]
@VolatilePulse wants to use `displayedStats` instead of `usedStats`.

**Comment by VolatilePulse** [comment]
Partially resolved with 1d934a8799a1432ed5f068868a96e820c1a78f26

**Comment by cadon** [comment]
As VolatilePulse suggested, `displayedStats` is more usefull than `usedStats`. Not used stats, i.e. stats that will not receive any levelup in wild or domesticated creatures, can just have a base-value of `0` and will be ignored totally.
So far the parameter `doesNotUseOxygen` was used to indicate a species that don't display its oxygen stat, i.e. aquatic species. The issue is, these species usually use this stat and also apply levels to it, so the stat is not not used but just hidden.

In conclusion, the needed parameter is `displayedStats`, the format is preferably a bit-array, stored as a 12-bit integer.

**Comment by VolatilePulse** [comment]
As seen in the `/Game/PrimalEarth/UI/Inventory/CharacterStatsPanel` asset, we can see the order of the stats how they would be represented in game if Wildcard decides to implement the other stats that creatures do not yet use. I was unable to locate any logic behind how stat bars are hidden, but it seems that we've uncovered the necessary parameters for hiding oxygen and all other stats are hidden purely under the condition of `DontUseValue` being 1 or 0. Until further evidence can be provided, I believe this is a "good enough" solution and can be evaluated for any issues.

![image](https://user-images.githubusercontent.com/6995232/62000646-023d2000-b0aa-11e9-8004-227d9721a941.png)
